### ISSUE TITLE
strict rule not finding "use strict"

### ISSUE THREAD

**Issue by boneskull** [eslint/eslint#4630]
ESLint v1.10.3

`.eslintrc`:

```yaml
env:
  node: true
  es6: true
parser: babel-eslint

rules:
  # errors
  comma-dangle: 2
  no-cond-assign:
    - 2
    - except-parens
  no-console: 0
  no-constant-condition: 2
  no-control-regex: 2
  no-debugger: 2
  no-dupe-args: 2
  no-dupe-keys: 2
  no-duplicate-case: 2
  no-empty-character-class: 2
  no-empty: 2
  no-ex-assign: 2
  no-extra-boolean-cast: 2
  no-extra-parens: 2
  no-extra-semi: 2
  no-func-assign: 2
  no-inner-declarations: 2
  no-invalid-regexp: 2
  no-irregular-whitespace: 2
  no-negated-in-lhs: 2
  no-obj-calls: 2
  no-regex-spaces: 2
  no-sparse-arrays: 2
  no-unreachable: 2
  use-isnan: 2
  valid-jsdoc:
    - 1
    -
      requireReturn: false
  valid-typeof: 2
  no-unexpected-multiline: 2

  # best practices
  accessor-pairs: 0
  block-scoped-var: 2
  complexity: 0
  consistent-return: 0
  curly:
    - 2
    - all
  default-case: 2
  dot-notation: 2
  dot-location:
    - 2
    - property
  eqeqeq: 2
  guard-for-in: 1
  no-alert: 2
  no-caller: 2
  no-div-regex: 2
  no-else-return: 2
  no-empty-label: 2
  no-eq-null: 2
  no-eval: 2
  no-extend-native: 2
  no-extra-bind: 2
  no-fallthrough: 2
  no-floating-decimal: 2
  no-implicit-coercion: 1
  no-implied-eval: 2
  no-invalid-this: 0
  no-iterator: 2
  no-labels: 1
  no-lone-blocks: 2
  no-loop-func: 2
  no-multi-spaces: 2
  no-multi-str: 2
  no-native-reassign: 2
  no-new-func: 2
  no-new-wrappers: 2
  no-new: 2
  no-octal-escape: 2
  no-octal: 2
  no-param-reassign: 0
  no-process-env: 1
  no-proto: 2
  no-redeclare: 2
  no-return-assign: 0
  no-script-url: 2
  no-self-compare: 2
  no-sequences: 2
  no-throw-literal: 2
  no-unused-expressions: 2
  no-useless-call: 2
  no-void: 2
  no-warning-comments: 0
  no-with: 2
  radix: 2
  vars-on-top: 2
  wrap-iife: 2
  yoda: 2

  # strict mode
  strict:
    - 2
    - global

  # variables
  init-declarations: 0
  no-catch-shadow: 0
  no-delete-var: 2
  no-label-var: 2
  no-shadow-restricted-names: 2
  no-shadow: 1
  no-undef-init: 2
  no-undef: 2
  no-undefined: 2
  no-unused-vars: 2
  no-use-before-define: 2

  # NodeJS
  callback-return: 0
  handle-callback-err: 2
  no-mixed-requires: 2
  no-new-require: 2
  no-path-concat: 2
  no-process-exit: 1
  no-restricted-modules: 0
  no-sync: 0

  # stylistic issues
  array-bracket-spacing:
    - 2
    - never
  brace-style:
    - 2
    - 1tbs
  camelcase:
    - 2
    -
      properties: always
  comma-spacing:
    - 2
    -
      before: false
      after: true
  comma-style:
    - 2
    - last
  computed-property-spacing:
    - 2
    - never
  consistent-this: 0
  eol-last: 2
  func-names: 1
  func-style:
    - 2
    - declaration
  id-length: 0
  indent:
    - 2
    - 2
  key-spacing:
    - 2
    -
      beforeColon: false
      afterColon: true
  lines-around-comment:
    - 2
    -
      beforeBlockComment: false
      beforeLineComment: false
  linebreak-style:
    - 2
    - unix
  max-nested-callbacks:
    - 2
    - 3
  new-cap: 0
  new-parens: 2
  newline-after-var: 0
  no-array-constructor: 0
  no-continue: 1
  no-inline-comments: 2
  no-lonely-if: 2
  no-mixed-spaces-and-tabs: 2
  no-multiple-empty-lines:
    - 2
    -
      max: 2
  no-nested-ternary: 2
  no-new-object: 2
  no-spaced-func: 2
  no-ternary: 0
  no-trailing-spaces: 2
  no-underscore-dangle: 0
  no-unneeded-ternary: 0
  object-curly-spacing:
    - 2
    - never
  one-var:
    - 2
    - never
  operator-assignment:
    - 2
    - always
  operator-linebreak:
    - 2
    - after
  padded-blocks:
    - 2
    - never
  quote-props: 0
  quotes: 0
  require-jsdoc: 0
  id-match: 0
  semi-spacing:
    - 2
    -
      before: false
      after: true
  semi:
    - 2
    - always
  sort-vars: 0
  space-after-keywords:
    - 2
    - always
  space-before-blocks:
    - 2
    - always
  space-before-function-paren:
    - 2
    - never
  space-in-parens:
    - 2
    - never
  space-infix-ops: 2
  space-return-throw-case: 2
  space-unary-ops: 2
  spaced-comment: 1
  wrap-regex: 0

  # es6
  arrow-parens:
    - 1
    - as-needed
  arrow-spacing: 2
  no-var: 2
  prefer-arrow-callback: 1
  prefer-const: 1
  prefer-template: 1

  # legacy
  max-len:
    - 2
    - 80
  max-params:
    - 2
    - 4
  no-bitwise: 1
  no-plusplus: 0
```

sample output:

```shell
$ npm test

> voltagedivider@0.0.0 test /Volumes/alien/projects/decipher/beacon/hermes/plugins/voltagedivider
> eslint . && karma start --single-run


/Volumes/alien/projects/decipher/beacon/hermes/plugins/voltagedivider/karma.conf.js
  3:1  error  Use the global form of "use strict"  strict

/Volumes/alien/projects/decipher/beacon/hermes/plugins/voltagedivider/src/index.js
  3:1  error  Use the global form of "use strict"  strict

/Volumes/alien/projects/decipher/beacon/hermes/plugins/voltagedivider/src/language/index.js
  3:1  error  Use the global form of "use strict"  strict

/Volumes/alien/projects/decipher/beacon/hermes/plugins/voltagedivider/src/language/language-picker/controller.js
  4:1  error  Use the global form of "use strict"  strict

/Volumes/alien/projects/decipher/beacon/hermes/plugins/voltagedivider/src/language/language-picker/index.js
  3:1  error  Use the global form of "use strict"  strict
```

sample file:

```js
'use strict';

const PickerCtrl = require('./controller');
const fs = require('fs');
const path = require('path');

function languagePicker() {
  return {
    restrict: 'E',
    template: fs.readFileSync(path.join(__dirname, 'language-picker.html'),
      'utf8'),
    controller: PickerCtrl
  };
}

module.exports = languagePicker;
```

Changing it to `"use strict";` (double-quotes) seems to have no effect.  This happened after an update from v1.9.0 to latest.

**Comment by eslintbot** [comment]
@boneskull Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by ilyavolodin** [comment]
Could you try it with default parser? It's possible that it's an issue with `babel-eslint`

**Comment by nuxdie** [comment]
@boneskull try use [babel-eslint@v4.1.6](https://github.com/babel/babel-eslint/releases/tag/v4.1.6) it helped me

**Comment by cbabos** [comment]
For me the problem is related to babel-eslint@5.x 
So I think it's only a problem with the new beta version, not with eslint itself. 
Thanks for the tips @boneskull and @ilyavolodin. Maybe the issue can be closed.

**Comment by boneskull** [comment]
this worked.  @nuxdie thx
